lib/child_process.js

```diff
@@ -93,16 +93,16 @@ exports._forkChild = function(fd) {
 };
 
 
-function normalizeExecArgs(command /*, options, callback*/) {
+function normalizeExecArgs(command, options_, callback_) {
   let options;
   let callback;
 
-  if (typeof arguments[1] === 'function') {
+  if (options_ && typeof options_ === 'function') {
```

Comment:
It's hard to know what V8 will do from version to version, but won't the leading `options_` require a conversion to a boolean and comparison against `true`?